notebooks/evaluation/rag_baseline_evaluation.ipynb

```diff
@@ -0,0 +1,3387 @@
+            "        return f\"{status_title} ({n_chunks_embedded / n_chunks:.0%})\"\n",
+            "    else:\n",
+            "        return status_title\n",
+            "\n",
+            "statuses = [pretty_upload_status(file[\"uuid\"], bearer_token) for file in file_list_response.json()]\n",
+            "statuses"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "-------"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "**Please ensure all emeddings have been completed before proceeding!**\n",
+            "\n",
+            "Keep calm and go for a tea break!"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "--------"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "#### Generate `actual_output` & `retrieval_context` (using RAG chat endpoint)"
+         ]
+      },
+      {
+         "cell_type": "code",
+         "execution_count": 21,
+         "metadata": {},
+         "outputs": [],
+         "source": [
+            "df = pd.read_csv(f'{V_SYNTHETIC}/ragas_synthetic_data.csv')\n",
+            "inputs = df['input'].tolist()"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "##### Using the RAG endpoint"
+         ]
+      },
+      {
+         "cell_type": "code",
+         "execution_count": 22,
+         "metadata": {},
+         "outputs": [],
+         "source": [
+            "df_endpoint = df.copy()\n",
+            "\n",
+            "retrieval_context = []\n",
+            "actual_output = []\n",
+            "\n",
+            "headers = {\n",
+            "    'accept': 'application/json',\n",
+            "    'Authorization': 'Bearer ' + bearer_token,\n",
+            "    'Content-Type': 'application/json',\n",
+            "}\n",
+            "\n",
+            "url = 'http://127.0.0.1:5002/chat/rag'\n",
+            "\n",
+            "for question in inputs:\n",
+            "    data = {\n",
+            "        \"message_history\": [\n",
+            "            {\n",
+            "                \"role\": \"user\",\n",
+            "                \"text\": question\n",
+            "            }\n",
+            "        ]\n",
+            "    }\n",
+            "    \n",
+            "    response = requests.post(url, headers=headers, data=json.dumps(data))\n",
+            "    data = response.json()\n",
+            "\n",
+            "    retrieval_context.append(data['source_documents'])\n",
+            "    actual_output.append(data['output_text'])\n",
+            "\n",
+            "df_endpoint['actual_output'] = actual_output\n",
+            "df_endpoint['retrieval_context'] = retrieval_context"
```

Comment:
Do we want to let people know they can keep an eye on progress via `docker compose logs -f core-api`?